app/assets/javascripts/application.js

```diff
@@ -77,4 +77,28 @@ $( document ).ready(function() {
       $(this).children(".plusminus").text('-');
     }
   });
-});
\ No newline at end of file
+});
+
+
+function hideNewsLetterValidation(email) {
+  if(email  != ""){
```

Comment:
@daryanypl 
<img width="1434" alt="Screen Shot 2020-10-09 at 1 17 26 PM" src="https://user-images.githubusercontent.com/30579702/95613088-62f81480-0a32-11eb-9719-9703530159b7.png">

Reply:
Email value is empty string only , not null.